server/ladders-local.ts

```diff
@@ -292,11 +288,17 @@ class LadderStore {
 
 			let reasons = '' + (Math.round(p1newElo) - Math.round(p1elo)) + ' for ' + (p1score > 0.9 ? 'winning' : (p1score < 0.1 ? 'losing' : 'tying'));
 			if (reasons.charAt(0) !== '-') reasons = '+' + reasons;
-			room.addRaw(Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; <strong>${Math.round(p1newElo)}</strong><br />(${reasons})`);
+			room.addRaw(
+				Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; ` +
+				Chat.html`<strong>${Math.round(p1newElo)}</strong><br />(${reasons})`
```

Comment:
What's your problem with doing it all on one line? It fits in 120 cols (I'm pretty sure) and I thought we carved out an exception in the line length limit for string literals anyway.

Reply:
It doesn't fit in 120 cols. https://i.imgur.com/M94ROln.png

Reply:
I guess I can disable it for those two lines?

Reply:
Oh, no, that's a flaw in my regex, then. You can leave this as-is.

Reply:
Reverted.

Reply:
Hey, wait, `Sanitize html in one line` passed linting! Your VS Code seems to not be picking up on my filter regex?

Reply:
Oh, that's,,, weird. I'll revert my reverted commit I guess.

Reply:
@Zarel this is still generating linter warnings.